packages/process/process.pony

```diff
@@ -32,7 +31,7 @@ actor Main
     // create a ProcessMonitor and spawn the child process
     let sp_auth = StartProcessAuth(env.root)
     let bp_auth = ApplyReleaseBackpressureAuth(env.root)
-    let pm: ProcessMonitor = ProcessMonitor(sp_auth, bp_auth, consume notifier,
+    let pm: ProcessMonitor = ProcessMonitor(sp_auth, bp_auth, consume client,
```

Comment:
also, not sure why we are doing `: ProcessMonitor` here. We have some annotations on some and none on others. I think we should either have none or all for the example, not this in between state.

Reply:
Well, you know that my preference is to annotate everywhere - (especially in docs where the purpose is to communicate as easily as possible).

The pony style-guide doesn't mention omitting types and refcaps on variable definitions so I didn't consider that to be an overriding concern...

Reply:
Then I would say, annotations for all here. And updates to commit message (if you update first message here and pr title, they can be used on squash).